pom.xml

```diff
@@ -202,6 +202,11 @@
             <version>6.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.vladmihalcea</groupId>
+            <artifactId>hibernate-types-5</artifactId>
+            <version>2.17.1</version>
```

Comment:
This is quite an old version of this library: https://github.com/vladmihalcea/hypersistence-utils has it with a new name and up to version 3.5